unstructured/metrics/evaluate.py

```diff
@@ -187,3 +173,25 @@ def measure_element_type_accuracy(
     _write_to_file(export_dir, "all-docs-element-type-frequency.tsv", df)
     _write_to_file(export_dir, "aggregate-scores-element-type.tsv", agg_df)
     _display(agg_df)
+
+
+def group_text_extraction_acc(
```

Comment:
What does `acc` stand for? Could be accumulation, accumulator, accessory, access, etc.

Can we just spell it out? I'm thinking it's `_accuracy`, but especially since it's a call now it improves readability if the user doesn't need to visit the implementation to make sense of the call.

Reply:
We'll want to call it through the metrics evaluation script

Reply:
@Klaijan not getting your comment. I'm suggesting you change the name to `group_text_extraction_accuracy()` (if "accuracy" is what is meant by "acc". What does that have to do with being called via the metrics evaluation script?